platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -814,7 +814,7 @@ enum tfm_plat_err_t init_debug(void)
 #error "Debug access controlled by NRF_APPROTECT and NRF_SECURE_APPROTECT."
 #endif
 
-#if defined(NRF_APPROTECT)
+#if defined(NRF_APPROTECT) && !defined(NRF54L15_XXAA)
```

Comment:
How about checking e.g. for `NRF_UICR_S` instead?

Reply:
Or some register/define in particular under UICR (`APPROTECT`, `SECUREAPPROTECT`).

Reply:
I think that we should use the devices as we are using them elsewhere. This is maybe something that could be changed when we know how the nRF54L15 is going to look. We want it to lock debugger permanently, but it is not supported in 2.8.

Reply:
though we are using `defined(NRF_APPROTECT)` here already :thinking: